mlrun/model_monitoring/db/stores/v3io_kv/kv_store.py

```diff
@@ -89,16 +89,20 @@ class SchemaParams:
 _EXCLUDE_SCHEMA_FILTER_EXPRESSION = '__name!=".#schema"'
 
 
-class KVStoreBase(mlrun.model_monitoring.db.StoreBase):
+class KVStoreBase(StoreBase):
+    type: str = mm_schemas.ModelEndpointTarget.V3IO_NOSQL
     """
     Handles the DB operations when the DB target is from type KV. For the KV operations, we use an instance of V3IO
     client and usually the KV table can be found under v3io:///users/pipelines/project-name/model-endpoints/endpoints/.
     """
 
-    def __init__(self, project: str, access_key: typing.Optional[str] = None) -> None:
+    def __init__(
+        self,
+        project: str,
+    ) -> None:
         super().__init__(project=project)
         # Initialize a V3IO client instance
-        self.access_key = access_key or os.environ.get("V3IO_ACCESS_KEY")
+        self.access_key = os.environ.get("V3IO_ACCESS_KEY")
```

Comment:
This can probably be safely removed in all places inside this class. This env var is taken by default by the clients.